mithril-common/src/crypto_helper/era.rs

```diff
@@ -1,18 +1,18 @@
-use ed25519_dalek::{ExpandedSecretKey, SignatureError};
-use rand_chacha_dalek_compat::rand_core::{self, CryptoRng, RngCore, SeedableRng};
-use rand_chacha_dalek_compat::ChaCha20Rng;
+use ed25519_dalek::{SignatureError, Signer};
```

Comment:
Are there reservations to importing the `ExpandedSecret` from the hazmat crate?

Reply:
I'm not sure I understand what you mean by that :sweat_smile: .
With this PR we suppress all usages of the `ExpandedSecretKey` (we don't even enable the `hazmat` feature of the `ed25519-dalek` crate).

Reply:
Ah got it thanks!! I was trying to upgrade without moving too much around / changing structure.